workflow-templates/go-check.yml

```diff
@@ -6,8 +6,16 @@ jobs:
     name: Go checks
     steps:
       - uses: actions/checkout@v2
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@d9f0e73c0497685d68af8c58280f49fcaf0545ff # v2.5.1
+      - uses: actions/setup-go@v2
         with:
-          version: v1.33
-          args: "--disable-all --enable gofmt,govet"
+          go-version: "1.16.x"
+      - name: Install staticcheck
+        run: go install honnef.co/go/tools/cmd/staticcheck@be534f007836a777104a15f2456cd1fffd3ddee8 # v2020.2.2
+      - name: go vet
+        run: go vet ./...
+      - name: staticcheck
+        if: ${{ success() || failure() }}
```

Comment:
I think this line is a no-op?

Reply:
Usually, if one step returns an error, GitHub Actions aborts the workflow run. I should probably add a comment `run this step even if the previous one failed`.

Reply:
Ah, gotcha. Yeah, I would have never guessed that, so a comment would help.